store/index.js

```diff
@@ -199,8 +199,8 @@ export const actions = {
                 category: "Diagnosis",
                 dataType: "string",
                 explanation: "This is an explanation for how to annotate diagnosis.",
-                options: { mode: "row" },
-                specializedComponent: "annot-vocabulary-row"
+                options: {},
+                specializedComponent: "annot-vocabulary"
```

Comment:
This guy doesn't exist anymore - throws a Vue warning in the console. Did you mean `annot-tool-group` here?

Reply:
@surchs Resolved?